src/System.Runtime.Serialization.Formatters/src/project.json

```diff
@@ -0,0 +1,24 @@
+{
+  "frameworks": {
+    "netstandard1.5": {
+      "dependencies": {
+        "Microsoft.NETCore.Platforms": "1.0.1-rc3-24026-00",
+        "System.Collections": "4.0.10",
+        "System.Diagnostics.Debug": "4.0.10",
+        "System.Resources.ResourceManager": "4.0.0",
+        "System.Runtime": "4.0.20",
+        "System.Runtime.Extensions": "4.0.10",
+        "System.Runtime.Serialization.Primitives": "4.1.1-rc3-24026-00",
+        "System.Threading.Tasks": "4.0.10"
+      },
+      "imports": [
+        "dotnet5.6"
+      ]
+    },
+    "net462": {
+      "dependencies": {
+        "Microsoft.TargetingPack.NETFramework.v4.6.1": "1.0.1"
```

Comment:
If you are intending this to only be net462 (which I think we should consider lowering) then you should use the v4.6.2 package.